tests/models/test_restore.py

```diff
@@ -270,42 +271,45 @@ def test_model_saving_loading(tmpdir):
     assert torch.all(torch.eq(pred_before_saving, new_pred)).item() == 1
 
 
-def test_load_model_with_missing_hparams(tmpdir):
-    trainer_options = dict(
-        progress_bar_refresh_rate=0,
-        max_epochs=1,
-        checkpoint_callback=ModelCheckpoint(tmpdir, save_top_k=-1),
-        logger=False,
-        default_root_dir=tmpdir,
-    )
-
-    # fit model
-    trainer = Trainer(**trainer_options)
-
-    class CurrentModelWithoutHparams(EvalModelTemplate):
-        def __init__(self):
-            super().__init__()
-
-    class CurrentModelUnusedHparams(EvalModelTemplate):
-        def __init__(self, hparams):
-            super().__init__()
+# def test_load_model_with_missing_hparams(tmpdir):
```

Comment:
did we want to remove this?

Reply:
this is absolute as we expand all vars in `hparams`

Reply:
I just wanted to point out that if we're going to comment this out, we should just remove it. i think you meant to say obsolete, which tells me that you actually want to remove this test altogether, not just comment it out.

Reply:
raising this again, did you mean to remove this code completely rather than just commenting it out?